src/components/forms/SearchBar/SearchBar.stories.tsx

```diff
@@ -24,18 +24,24 @@ playground.parameters = {
   chromatic: { disable: true },
 };
 playground.argTypes = {
+  hasSuggestions: { control: 'boolean' },
   isDisabled: { control: 'boolean' },
   isInvalid: { control: 'boolean' },
 };
 
-export const DefaultSearchBar: Story = () => (
-  <SearchBar aria-label="SearchBar" {...commonArgs} />
+export const Default: Story = () => (
+  <SearchBar aria-label="SearchBar" hasSuggestions {...commonArgs} />
```

Comment:
I would prefer no to add `hasSuggestions` prop. When you say it's `Default` story I would expect that it shows the state with an absolute minimum required configuration.